control-plane/roles/gardener/templates/etcd-values.j2

```diff
@@ -9,6 +9,12 @@ backup:
   storageProvider: "GCS"
   gcs:
     serviceAccountJson: {{ gardener_backup_infrastructure_service_account_json | to_json }}
+{% elif gardener_backup_infrastructure.provider == "S3" %}
```

Comment:
Can we unify this by changing line 8 to `{% if gardener_backup_infrastructure.provider == "gcp" %}`? I checked for metalstack.cloud and it would work. However we need to add a breaking change to PR if we do it for people to check.